apps/aechannel/src/aesc_create_tx.erl

```diff
@@ -152,7 +152,8 @@ process(#channel_create_tx{} = Tx, Trees, Env) ->
           state_hash(Tx),
           lock_period(Tx),
           fee(Tx),
-          nonce(Tx)),
+          nonce(Tx),
+          round(Tx)),
```

Comment:
I see this is [hardcoded to 1](https://github.com/aeternity/aeternity/blob/8f04de354e9a85141636cee68e2c9415803ec1d3/apps/aechannel/src/aesc_create_tx.erl#L308). Is the plan to include the round in the transaction? Because this syntax suggests that the round is a variable in the transaction serialized by the user though it is not.

Reply:
No, there is no plan on doing it. The `round/1` callback is needed so we don't have special processing for the `channel_create_tx` in the FSM.
This change is just for consisteny.